vet.sh

```diff
@@ -37,8 +37,8 @@ if [ "$1" = "-install" ]; then
     if [[ "$TRAVIS" = "true" ]]; then
       PROTOBUF_VERSION=3.3.0
       cd /home/travis
-      wget https://github.com/google/protobuf/releases/download/v$PROTOBUF_VERSION/$basename-linux-x86_64.zip
-      unzip $basename-linux-x86_64.zip
+      wget https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/protoc-${PROTOBUF_VERSION}-linux-x86_64.zip
+      unzip protoc-linux-x86_64.zip
```

Comment:
:man_facepalming: version is missed here.

And make `$check_proto` true so travis will test it?

Reply:
Derp.

Are you saying to set check_proto temporarily to test this PR?